ansible/roles/formplayer/templates/sentry.properties.j2

```diff
@@ -0,0 +1,5 @@
+sentry.dsn={{ formplayer_sentry_dsn }}
+{% if http_proxy_address is defined %}
+http.proxy.host={{ http_proxy_address }}
+http.proxy.port={{ http_proxy_port }}
+{% endif %}
```

Comment:
What is `sentry.properties.j2`? Is it used somewhere?

Or, from the other side, what do you want to end up on the server / in the formplayer directory?